### ISSUE TITLE
Outland chimera as hunter pet has boars ability?

### ISSUE THREAD

**Issue by Kirishiki** [dalaranwow/dalaran-wow#5288]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Tamed this green chimera pet from blades edge mountain and it has ability called Gore "your boar gores the target bal bla bla, does double dmg if used after charge" (what chimeras dont have) 
**Expected behaviour**: Tell us what should happen instead
well i guess it shouldnt have it lol
**Steps to reproduce the problem**:

1. grab all of them green chimeras.
2. remove the ability with uber gm powers.
3. release them bak to wild.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
http://tinypic.com/r/339io80/9

**Comment by AttendantTock** [comment]
Hello,

I can confirm that on the current PTR, this Bladewind Bloodletter from the Blade's Edge Mountains has an extra ability (Gore rank 2).

According to https://wotlk.evowow.com/?npc=21033#controlled-abilities there should only be Bite and Froststorm Breath.

<img width="614" alt="capture d ecran 2018-04-13 a 20 06 49" src="https://user-images.githubusercontent.com/29557022/38753575-b67282f2-3f56-11e8-9570-edee859f3907.png">
